include/onnxruntime/core/common/logging/capture.h

```diff
@@ -4,7 +4,7 @@
 #pragma once
 
 #include <cstdarg>
-#include "core/common/gsl.h"
+#include <gsl/gsl>
```

Comment:
I think it does not count as a system header, so the style guide says to use quotes instead of angle brackets.
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes
```suggestion
#include "gsl/gsl"
```

Reply:
The difference between "" and <> is the search order.  <> prefers the folders added  by -Ixxx, while "" perfers the current folder

Reply:
See: https://gcc.gnu.org/onlinedocs/cpp/Search-Path.html

The quote form of the directive #include "file" is meant to be used for searching files that are relative to the current directory. In all other cases, we should use angle-bracket instead.